add token sdk sample application

arner opened the pull request:
This is a service with a REST API that wraps the [Token SDK](https://github.com/hyperledger-labs/fabric-token-sdk) to issue, transfer and redeem tokens backed by a Hyperledger Fabric network for validation and settlement.

denyeart commented on token-sdk/README.md:
I think the largest missing piece is an explanation of what happened in each of the transactions.
- which nodes had to collaborate?
- what logic is in the FSC flow in the nodes versus in the chaincode on the Fabric peers?
- what data is stored on the nodes after each transaction?
- what data is stored on the Fabric ledger after each transaction?
- how is privacy maintained?
- what does the auditor do?

denyeart commented on token-sdk/scripts/enroll-users.sh:
```diff
@@ -0,0 +1,39 @@
+# Fabric Smart Client node identities (identity of the node, used when talking to other nodes)
+for node in issuer auditor owner1 owner2
+do
+  fabric-ca-client register --id.name fsc${node} --id.secret password --id.type client
```
For each of these register commands I had to add `-u http://localhost:27054`

adecaro commented:
@denyeart , this looks good to me, thanks :)

arner commented:
Thanks for the review! Updated the readme to implement your feedback. Also added configuration for the blockchain explorer to show what goes on the ledger.

denyeart commented on token-sdk/scripts/enroll-users.sh:
```diff
@@ -0,0 +1,39 @@
+fabric-ca-client register -u http://localhost:27054 --id.name alice --id.secret password --id.type client --enrollment.type idemix --idemix.curve gurvy.Bn254
+fabric-ca-client enroll -u http://alice:password@localhost:27054  -M "$(pwd)/keys/owner1/wallet/alice/msp" --enrollment.type idemix --idemix.curve gurvy.Bn254
+
+fabric-ca-client register --id.name bob --id.secret password --id.type client --enrollment.type idemix --idemix.curve gurvy.Bn254
```
Isn't `-u http://localhost:27054` needed here too?

denyeart commented on token-sdk/README.md:
900 right?

denyeart commented on token-sdk/README.md:
Isn't it a transfer of 100 EURX to bob? Although maybe you want to describe a node 2 scenario...

arner replied:
Good point, I changed the curl commands to also transfer 100 TOK to dan to align the scenarios.
